### DISCUSSION TITLE
All cams in m3u8 list

### DISCUSSION THREAD

**Discussion by nlfog** [mrlt8/docker-wyze-bridge#653] [General]
My goal here is to be able to add my cameras as a playlist to add to Tivimate on Android TV so I can view cameras in a multi-view function while watching TV.  I have tried creating my own m3u file and adding in the HLS links for each camera, but that failed.  I tried using just the one camera and adding it (http://192.168.1.2:8888/camera/stream.m3u8) and tivimate finds it, but doesn't see any channels in there.  How can I create an m3u8 file that has all my cameras in it?

**Comment by mrlt8** [comment]
Good question. Would something like this work as an m3u8?

```m3u8
#EXTM3U

#EXTINF:-1 channel-id="WYZECAM1" channel-number="1" tvc-guide-genres="wyze" tvc-guide-tags="Live" tvg-logo="http://localhost:5000/snapshot/front.jpg", WYZECAM1
http://localhost:8888/front/stream.m3u8

#EXTINF:-1 channel-id="WYZECAM2" channel-number="2" tvc-guide-genres="wyze" tvc-guide-tags="Live" tvg-logo="http://localhost:5000/snapshot/side.jpg", WYZECAM2
http://localhost:8888/side/stream.m3u8
```

**Reply by nlfog** [reply]
That worked beautifully!  I now have my cameras listed in Tivimate and able to setup multi-view.  Thank you!
